conda_forge_tick/migrators.xsh

```diff
@@ -1076,3 +1076,55 @@ class BlasRebuild(Rebuild):
     def pr_title(self):
         return 'Rebuild for new BLAS scheme'
 
+
+class RBaseRebuild(Rebuild):
+    """Migrator for rebuilding all R packages."""
+    migrator_version = 0
+    rerender = True
+    bump_number = 1
+
+    def migrate(self, recipe_dir, attrs, **kwargs):
+        # Set the provider to Azure only
+        with indir(recipe_dir + '/..'):
+            if os.path.exists('conda-forge.yml'):
+                with open('conda-forge.yml', 'r') as f:
+                    y = safe_load(f)
+            else:
+                y = {}
+            if 'provider' not in y:
+                y['provider'] = {}
+            y['provider']['win'] = 'azure'
+            with open('conda-forge.yml', 'w') as f:
+                safe_dump(y, f)
+
+            with open('meta.yaml', 'r') as f:
+                text = f.read()
+
+            changed = False
+            lines = text.split("\n")
+            
+            if attrs['feedstock_name'].startswith("r-") and "- conda-forge/r" not in text \
+                    and any(a in text for a in ["johanneskoester", "bgruening", "daler", "jdblischak", "cbrueffer", "dbast", "dpryan79"]):
+                for i, line in enumerate(lines):
+                    if line.strip() == "recipe-maintainers:" and i + 1 < len(lines):
+                        lines[i] = line + "\n" + lines[i+1][:lines[i+1].index("-")] + "- conda-forge/r"
```

Comment:
This would fail in case a blank/empty or comment line is at the start of the maintainer list.
Thankfully, this doesn't seem to be common:
```sh
|> grep -A1 '^ *recipe-maintainers:' */recipe/meta.yaml | grep -E 'yaml- *($|#)'
oq-engine/recipe/meta.yaml-    # GitHub IDs for maintainers of the recipe.
pytest-mccabe/recipe/meta.yaml-    # GitHub IDs for maintainers of the recipe.
```
and no `r-*` recipes are affected.
I'm not familiar on how the migrators are handled -- if it's just expected to run once, i.e., on the current set of feedstocks, then you can disregard my comment since no `r-*-feedstock` is affected.

Reply:
It's run only on R recipes, so this should be fine